### PR TITLE
Contrib/concentric ai -> Playbook enhancement.

### PR DESCRIPTION
**Pull request by shams-concentricai**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [] In Progress
- [X] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
This PR hopes to improve performance and enhance the playbook of the existing integration.

## Minimum version of Demisto
- [ ] 5.0.0
- [ ] 5.5.0
- [X ] 6.0.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [X] Tests
- [ ] Documentation

### CODE REVIEW

**Comment by shams-concentricai** [comment]
Hi Team 

I am hoping to get this resolves soon ,please let me know if anything else is required. 
Also i have some doubts , would love to clarify some of em with you.

**Comment by shams-concentricai** [comment]
Hi @teizenman can you help me with the circleci validations , it's claiming that something is wrong with my release notes but not mentioning what exactly !

**Comment by shams-concentricai** [comment]
Hi @teizenman  , could you please review this ?

File: Packs/ConcentricAI/ReleaseNotes/1_2_0.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+#### Mappers
+##### New: ConcentricAi-mapper
+- %%UPDATE_RN%%
+
+#### Classifiers
+##### New: ConcentricAi
+- Incoming classifier of incidents into demisto
+
+#### Incident Types
+- **ConcentricAI Security Event**- Updated the Docker image to: *demisto/python3:3.9.1.15759*.
```
Comment (teizenman) [inline Packs/ConcentricAI/ReleaseNotes/1_2_0.md:18]: Those items were not changed, please remove them from the RN file.

**Review by teizenman — CHANGES_REQUESTED** [review]
Please do not delete the content of the 1_1_0.md file.
You should just update the new RN file 1_2_0.md.

**Comment by teizenman** [comment]
> Please do not delete the content of the 1_1_0.md file.
> You should just update the new RN file 1_2_0.md.

@shams-concentricai Please note this comment as well.

**Comment by shams-concentricai** [comment]
@teizenman the checks are passing now , could you please approve ?

File: Packs/ConcentricAI/ReleaseNotes/1_2_0.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+#### Integrations
+##### ConcentricAI
+- Added enhancement to playbook and overall functioning.
+#### Playbooks
+##### ConcentricAI Demo Playbook
+- The Playbook now enables one to fetch all file details, file sharing permissions
+     ,user details of the owner along with summaryReport of the incident.
```
Comment (teizenman) [inline Packs/ConcentricAI/ReleaseNotes/1_2_0.md:7]:
```suggestion
- The Playbook now enables one to fetch all file details, file sharing permissions, user details of the owner along with summaryReport of the incident.
```

I guess that the new line was made by mistake: 
image

**Review by teizenman — CHANGES_REQUESTED** [review]

**Comment by teizenman** [comment]
@shams-concentricai 
I will merge it after you will push the fix for the RN.

**Review by teizenman — APPROVED** [review]

**Pull request merged**